tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +188,22 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_and_add_new_user(self):
+        response = self.client.get(reverse("admin:auth_group_add"))
+        self.assertContains(response, "<h1>%s</h1>" % (_("Add %s") % _("group")))
+        self.assertContains(
+            response,
+            '<label for="id_users">%s</label>' % _('Users')
+        )
+
+    def test_should_see_a_user_add_option_while_edit_a_group(self):
+        response = self.client.get(
+            reverse("admin:auth_group_change", args=[self.group.id])
+        )
+
+        self.assertContains(response, "User0")
```

Comment:
This assertion fails b/c self.group doesn't contain any users probably.

In addition the assertion needs to be more specific. As-is it will pass in all cases then the text "User0" appears on the page. You want to make sure this text is in the right place (e.g. left-hand side selector and/or right hand side selector if user is already in the group).

Reply:
Note: "User0" is undefined. That should be a specific object from the User class !

Reply:
Still working on this one